src/test/java/com/thealgorithms/maths/HeronsFormulaTest.java

```diff
@@ -24,4 +24,9 @@ void test3() {
     void test4() {
         Assertions.assertEquals(HeronsFormula.Herons(4, 5, 8), 8.181534085976786);
     }
+
+    @Test
+    public void testCalculateAreaWithInvalidInput() {
+        Assertions.assertThrows(IllegalArgumentException.class, () -> { HeronsFormula.Herons(1, 2, 3); });
```

Comment:
This will give _100%_ coverage.

```suggestion
        Assertions.assertThrows(IllegalArgumentException.class, () -> { HeronsFormula.Herons(1, 2, 3); });
        Assertions.assertThrows(IllegalArgumentException.class, () -> { HeronsFormula.Herons(2, 1, 3); });
        Assertions.assertThrows(IllegalArgumentException.class, () -> { HeronsFormula.Herons(3, 2, 1); });
```